src/lib/infrastructure/repository/arangodb-migrate.ts

```diff
@@ -0,0 +1,12 @@
+import { Database } from 'arangojs'
+import * as fs from 'fs'
+import * as path from 'path'
+
+export async function migrate(db: Database) {
+  const migrationFiles = fs.readdirSync(path.join(__dirname, 'arangodb-migrations')).sort()
```

Comment:
if sorting by name, then the migration file naming should not start with a date in dd/mm/yyyy format. You could use a yyyy/mm/dd date format instead.